cmd/lncli/commands.go

```diff
@@ -2520,11 +2528,14 @@ func getNodeInfo(ctx *cli.Context) error {
 }
 
 var queryRoutesCommand = cli.Command{
-	Name:        "queryroutes",
-	Category:    "Payments",
-	Usage:       "Query a route to a destination.",
-	Description: "Queries the channel router for a potential path to the destination that has sufficient flow for the amount including fees",
-	ArgsUsage:   "dest amt",
+	Name:     "queryroutes",
+	Category: "Payments",
+	Usage:    "Query a route to a destination.",
+	Description: `
+			Queries the channel router for a potential path to the destination that has sufficient flow for 
+			the amount including fees. By default the fee limit capped at the send amount.
```

Comment:
Capped at send amount, is this implemented? I think the default is no cap?

Reply:
It defaults to the amt sent in the rpc server. https://github.com/lightningnetwork/lnd/blob/4c0ca3717423367be59973928b5ff908ff260259/rpcserver.go#L1969